app/helpers/indicators_helper.rb

```diff
@@ -26,4 +26,8 @@ def get_buy_sell_class(value)
       'neutral'
     end
   end
+
+  def github_url(repo)
+    "https://www.github.com/#{repo}"
```

Comment:
I think their default is to not include the `www` subdomain so might be good to remove it here as well.